_component_design/input-selectors.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Input Selectors
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  An input selector enables the user to select one value from a list. When the input selector is inactive, it displays a single value. When the input selector is activated by the user, it displays a list of values from which the user can choose.
+preview-image: preview-images/drop-downs.svg
+resource: true
+status: stable
+need: selectors
```

Comment:
Line not needed; should be removed (this k:v pair is to trigger custom javascript and really is only needed on one or two pages).

Reply:
Done.